tests/functional/locators.py

```diff
@@ -59,12 +59,17 @@ def lowest_level_of_geography_radio_button(index_value):
     @staticmethod
     def frequency_radio_button(index_value):
         # index_value should be in the range 0 to 6
-        return (By.ID, "frequency_id-%s" % str(index_value))
+        return (By.ID, "data-source-1-frequency_of_release_id-%s" % str(index_value))
+
+    @staticmethod
+    def type_of_data_checkbox(index_value):
+        # index_value should be in the range 0 to 1
+        return (By.ID, "data-source-1-type_of_data-%s" % str(index_value))
 
     @staticmethod
     def type_of_statistic_radio_button(index_value):
         # index_value should be in the range 0 to 6
```

Comment:
Do we know *why* it should be between 0 and 6?  We only have 4 entries in the `type_of_statistic` table. What are the extra three possible indexes for?  If not then let's delete the comment.

Reply:
This is an old comment I think. I'll remove it.

Reply:
Updated them instead, with references to where they come from.